cluster/src/main/java/org/apache/iotdb/cluster/log/catchup/LogCatchUpTask.java

```diff
@@ -50,8 +52,9 @@
 
   // sending logs may take longer than normal communications
   private static final long SEND_LOGS_WAIT_MS = 5 * 60 * 1000L;
+  private static final int LOG_NUM_IN_BATCH = 1024;
+  private static final int LEFT_SIZE_IN_REQUEST = 1024;
```

Comment:
These variables seems to be easier to maintain and modify in the clusterUtil or clusterConfig